packages/carbon-web-components/src/globals/CarbonBase.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright IBM Corp. 2016, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// @ts-nocheck
+import { LitElement } from 'lit';
+import CarbonStyles from '../internal/vendor/@carbon/ibmdotcom-utilities/utilities/CarbonStyles/CarbonStyles';
+
+export default class CarbonBase extends LitElement {
+  carbonStyles?: String[];
```

Comment:
It would be neat if we could type this more narrowly, so that when you consume it, you get auto-complete on the ones presently available. Not totally sure, but somewhere in the utilities package it would export a type definition, that we could consume here.

From `carbonStyles.js`

```
const importedStyleSheets = {
  buttonStyles,
  resetStyles,
  typeStyles,
};

type ImportedStyleSheets = keyof importedStyleSheets;
```

Now this line would become:

```
carbonStyles?: ImportedStyleSheets[];
```

Naming sucks, but hoepfully you get the idea?

Definately in the category of "Bonus"